migrate-configuration-v2.rst

```diff
@@ -0,0 +1,90 @@
+.. post:: May 31, 2023
+   :tags: builders, deprecation
+   :author: Manuel
+   :location: BCN
+   :category: Changelog
+
+Migrate your project to ``.readthedocs.yaml`` configuration file v2
+===================================================================
+
+We are announcing the **deprecation of building without a configuration file** (``.readtehdocs.yaml``)
+together with the deprecation of configuration file v1 as well.
+
+Read the Docs *will soon start requiring* a ``.readthedocs.yaml`` configuration file
+for all the projects to build their documentation successfully.
+We will stop creating a default configuration file *behind the scenes*, making assumptions for your projects
+and trying our best to make the builds succeed.
+
+We plan to deploy these changes and **start failing builds without a configuration file on September 2023**.
+This matches the period when ``urllib3`` `drops support for OpenSSL 1.1.1 <https://github.com/urllib3/urllib3/issues/2168>`_
+which is OpenSSL version that Ubuntu 18.04 has installed and it's used when the project doesn't have a configuration file.
+
+The main reason behind this decision is it's pretty hard to maintain a set of defaults that are useful
+for the amount of users we have and the number of different tools they use to build documentation.
+Even when the defaults we chose were useful for a period of time,
+we found they were almost impossible to update to newer versions of dependencies
+because doing so would break a lot of projects that weren't using a configuration file
+and pinning their dependencies (e.g. OS, Python version, dependencies, etc).
+
+If you have a project on Read the Docs that it's not using a ``.readthedocs.yaml`` configuration file,
+**we strongly recommend you to create one as soon as possible** to avoid breaking builds once we deploy the changes.
```

Comment:
```suggestion
Add your first .readthedocs.yaml

If you have a project on Read the Docs that is not using a ``.readthedocs.yaml`` configuration file,
**we strongly recommend you to create one as soon as possible** to avoid breaking builds once we deploy the changes.
```